yaptide/persistence/models.py

```diff
@@ -269,6 +269,12 @@ class CeleryTaskModel(TaskModel):
     __tablename__ = 'CeleryTask'
     id: Column[int] = db.Column(db.Integer, db.ForeignKey('Task.id', ondelete="CASCADE"), primary_key=True)
     celery_id: Column[str] = db.Column(db.String, nullable=False, default="", doc="Celery task ID")
+    sim_pid: Column[int] = db.Column(db.Integer,
+                                     nullable=True,
+                                     doc='Id of simulation process used to communicate with shieldhit process')
+    path_to_sim: Column[str] = db.Column(db.String,
```

Comment:
maybe better to call that directory a workspace ?

Reply:
also - why do you keep that in the DB ? such directory is usually removed once the job is done